dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -127,6 +127,58 @@ def toggle_hidden
     render json: []
   end
 
+  # Provides an aggregated view of instructions for a script, to make things easier
+  # for levelbuilders
+  def instructions
+    require_levelbuilder_mode
+    authorize! :read, ScriptLevel
+
+    script = Script.get_from_cache(params[:script_id])
+
+    render 'levels/instructions', locals: { stages: script.stages }
+  end
+
+  # Provides a JSON summary of a particular stage
+  def summary
```

Comment:
It exposes a different set of information. I was struggling with this a little myself. Perhaps one solution would be to name this something like lesson_plan_summary and add a comment making it clear that this is what it's used for? Still leaves us in the unfortunate situation of having two different summaries though.

Incidentally, I also rather dislike `Script#summarize` and some of the stuff it passes down (for example icon-names).